app.rb

```diff
@@ -79,23 +78,33 @@
   @page_title = @term[:name]
 
   last_sha = @house[:sha]
-  csv_file = EveryPolitician::GithubFile.new(@term[:csv], last_sha)
-  @csv = CSV.parse(csv_file.raw, headers: true, header_converters: :symbol, converters: nil)
-
-  person_ids = @csv.map { |row| row[:id] }.uniq
 
   popolo_file = EveryPolitician::GithubFile.new(@house[:popolo], last_sha)
   popolo = EveryPolitician::Popolo.parse(popolo_file.raw)
-  people = popolo.persons.find_all { |p| person_ids.include?(p.id) }.sort_by { |p| p.sort_name }
 
-  @parties = @csv.find_all { |r| r[:end_date].to_s.empty? || r[:end_date] == @term[:end_date] }
-                 .group_by { |r| r[:group] }
-                 .sort_by { |p, ms| [-ms.count, p] }
+  memberships = popolo.memberships.find_all { |m| m.legislative_period_id.split('/').last == id }
+
+  person_ids = memberships.map(&:person_id)
```

Comment:
Probably doesn't make a huge amount of difference as this variable is only used once, but as these things have a habit of expanding over time, and just to be clear what's going on, it might be nice to `.uniq` this, as a Person might have more than one Membership during a term.